### PR TITLE
Jr2 88 import x28 synonyms

### PR DESCRIPTION
**Pull request by michael-wirth**

### CODE REVIEW

File: src/main/java/ch/admin/seco/service/reference/web/rest/OccupationResource.java [hunk 1]
```diff
@@ -89,15 +93,31 @@ public OccupationResource(OccupationService occupationService) {
...
         log.debug("REST request to update OccupationSynonym : {}", occupationSynonym);
-        if (occupationSynonym.getId() == null) {
-            return createOccupationSynonym(occupationSynonym);
+
+        if (isNull(occupationSynonym.getId())) {
+            if (!occupationService.findOneOccupationSynonymByExternalId(occupationSynonym.getExternalId())
+                .map(item -> {
+                    occupationSynonym.setId(item.getId());
+                    return occupationSynonym;
+                }).isPresent()) {
```
Comment (varusakov) [inline src/main/java/ch/admin/seco/service/reference/web/rest/OccupationResource.java]: Introduce local variable for statement inside if condition. It's hard to understand this expression.

File: src/main/resources/config/elasticsearch/mappings/locality.json [hunk 2]
```diff
@@ -2,7 +2,25 @@
     "localities": {
         "properties": {
             "city": {
-                "type": "text"
+                "type": "text",
+                "fields": {
+                    "de": {
```
Comment (varusakov) [inline src/main/resources/config/elasticsearch/mappings/locality.json]: Do we need to introduce language/translations(as in Occupation entity) fields to Locality model?

Comment (michael-wirth) [inline src/main/resources/config/elasticsearch/mappings/locality.json]: yes we want to search for Zürich and Zuerich and Zurich

Comment (michael-wirth) [inline src/main/resources/config/elasticsearch/mappings/locality.json]: Is there a better way to accomplish this?

Comment (varusakov) [inline src/main/resources/config/elasticsearch/mappings/locality.json]: No, I just want to clarify requirements

File: src/main/java/ch/admin/seco/service/reference/web/rest/OccupationResource.java (hunk 1 above)
Comment (michael-wirth) [inline src/main/java/ch/admin/seco/service/reference/web/rest/OccupationResource.java]: extracted expression into a private method

**Pull request merged**